src/sentry/static/sentry/app/views/projectInstall/platform.jsx

```diff
@@ -56,6 +57,10 @@ const ProjectInstallPlatform = React.createClass({
     }
   },
 
+  isGettingStarted() {
+    return location.href.indexOf('getting-started') > 0;
```

Comment:
I suppose this will naively fail if a user's organization or project is ever named `getting-started`.